.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: go
+dist: trusty
+
+go:
+  - 1.11.x
+
+install:
+  - go get -u golang.org/x/lint/golint
+
+before_script:
+  - set -e
+  - go list ./... | grep -v /proto/ | xargs -n 1 golint
+  - go tool vet  ./cmd/ ./pkg/
+  - git ls-files | grep -v proto |grep ".go$" | xargs gofmt -l | wc -l
+
+script:
+  - echo 0
```

Comment:
do you want to add uploading the docker image on master branch merge? Can use the credentials and stuff from here:
https://github.com/nimbess/nimbess-agent/blob/master/.travis.yml

I created the nimbess/stargazer repo for you in docker

Reply:
yes, will add then in a follow up patch. I think this image will change a bit once I have the etcd parts included.